### ISSUE TITLE
False-alarm in `TRIO210` - dicts, queues, etc are not sync http clients

### ISSUE THREAD

**Issue by Zac-HD** [Zac-HD/flake8-trio#129]
```python
async def f():
    r = {}
    r.get("not a sync http client")  # False alarm: raises TRIO210
```

I've seen similar issues on `Queue().put(...)`, `SomeCls.delete()`, etc.
